pages/common/mycli.md

```diff
@@ -13,3 +13,11 @@
 - Connect to a database on the specified host with the specified user:
 
 `mycli -u {{user}} -h {{host}} {{database_name}}`
+
+- Connect to a database with URI:
+
+`mycli mysql://{{user}}@{{host}}`
```

Comment:
We should stick with only one type of usage. The above examples show `-u` and `-h`. This seems to be just a short hand representation of it.

Reply:
It is hard to say which one is better, But people like me are often given the latter one because URI is easy to share and keep.

Reply:
Sure, I am not saying this is better or worse. I am just concerned about consistency. And since other pages like mysql and psql follow the existing format, I raised a concern regarding the benefits of changing such a convention.